src/components/shared/Terminal.tsx

```diff
@@ -104,15 +113,131 @@ function Terminal(prop: {
     return executeTouch(path, true);
   }
 
+  function executeRm(path: string, flags: string, directory = false): string[] {
```

Comment:
Add a comment above this function explaining what it does -- is it the same as `rm`? Does it remove directories? Etc. Just a high-level comment will suffice 🙂

Reply:
Yeah, I can see how that might be unclear now. Will be adding some commends to better explain on a high-level what these functions do.